.circleci/config.yml

```diff
@@ -41,8 +51,29 @@ initWorkingDir: &initWorkingDir
 
 jobs:
 
-  # Build all the templates, verify the templates work.
-  build:
+  # Build all the templates in one namespace, verify the templates work.
```

Comment:
That's now done by the 'build' target - it builds both the demo ( one ns ) and few other templates.
This is also used for 'kubectl apply -k"